docs/source/oci/advanced/logging.rst

```diff
@@ -0,0 +1,49 @@
+.. _oci_logging:
+
+Logging and Output
+==================
+
+Writing Custodian Logs to Object Storage
+-----------------------------------------
+
+You can output logs and resource records to Object Storage.
+By default, Custodian will add the policy name and date as the prefix to the blob.:
+
+    .. code-block:: sh
+
+        custodian run -s oci://<Bucket_Name>/<Optional_Folder_Name>/ <policy file>
```

Comment:
is folder (ui construct) in common use for oci users, versus prefix?

Reply:
Official documentation uses 'prefix'. Updated to match that